src/pages/Pool/index.tsx

```diff
@@ -174,6 +192,16 @@ export default function Pool() {
 
             <CTACards />
 
+            {positions?.some((p) => p.liquidity?.isZero()) ? (
+              <ShowInactiveToggle>
+                <TYPE.darkGray>{t('Hide closed positions')}</TYPE.darkGray>
```

Comment:
this is separate but we shouldn't use the english translation for translation keys (can be addressed with the larger task of improving i18n throughout the app), see https://github.com/Uniswap/uniswap-interface/pull/1026#issuecomment-675541852 for discussion

Reply:
Ack.